mariadb/Dockerfile

```diff
@@ -23,7 +23,10 @@ RUN --mount=type=cache,id=mariadb-apk-${TARGETARCH},sharing=locked,target=/var/c
 # base image. Set to 10 minutes just incase it ran on very old or overallocated
 # hardware.
 ENV S6_CMD_WAIT_FOR_SERVICES_MAXTIME=600000
-# Default Mariadb value of 16 MB (bytes)
-ENV MYSQL_MAX_ALLOWED_PACKET=16777216
+
+ENV \ 
+    # Default Mariadb value of 16 MB (bytes)
+    MYSQL_MAX_ALLOWED_PACKET=16777216 \
+    TRANSACTION_ISOLATION=READ-COMMITTED
```

Comment:
Please prefix `TRANSACTION_ISOLATION` with `MYSQL` to be consistent with the environmental naming conventions across the repository.

So `MYSQL_TRANSACTION_ISOLATION`.